templates/status.html

```diff
@@ -86,6 +88,47 @@
                 request_api(website);
             });
         }
+
+        function constructIconElement(website, css) {
+            // defaults to font awesome icons
+            var iconElement = {
+                attributeValue: `fa fa-${website}`,
+                htmlAttribute: 'class',
+                htmlElement: 'i'
+            };
+
+            if(css.hasOwnProperty(website)) {
+                // the key defined in the yml. eg: url, fontawesome
+                var ymlKey = Object.keys(css[website])[0];
+                // determines html attribute to be used to display icon. eg: src, class
+                iconElement.htmlAttribute = determineHtmlAttribute(ymlKey);
+                // determines html element to be used based off of the attribute. eg: src, class
+                iconElement.htmlElement = determineHtmlElement(iconElement.htmlAttribute);
+                // gets the attribute value from the yml file by using the attribute as the key
+                iconElement.attributeValue = css[website][ymlKey];
+            }
+            return iconElement;
+        }
+
+        function determineHtmlElement(htmlAttribute) {
+            var elementName;
+            if(htmlAttribute == 'src') {
```

Comment:
It's always a good habit to use `===` instead of `==` :P

Reply:
Also add a space before `(`

Reply:
Sure. A space before the `if` or a space before the function?